code/modules/antagonists/fugitive/fugitive_equipment.dm

```diff
@@ -0,0 +1,63 @@
+/obj/item/implant/camouflage
+	name = "experimental camouflage implant"
+	desc = "Allows its owner to blend in with their surroundings. Cool!"
+	actions_types = list(/datum/action/item_action/camouflage)
+
+/obj/item/implant/camouflage/emp_act(severity)
+	. = ..()
+
+	if(prob(15 * severity))
```

Comment:
should we have a `min()` here with some kind of default value (created as a define)

Reply:
alright wasn't sure